cranelift/interpreter/src/step.rs

```diff
@@ -977,35 +983,57 @@ pub enum CraneliftTrap {
 }
 
 /// Compare two values using the given integer condition `code`.
-fn icmp<V>(code: IntCC, left: &V, right: &V) -> ValueResult<bool>
+fn icmp<V>(_ty: types::Type, code: IntCC, left: &V, right: &V) -> ValueResult<V>
```

Comment:
Usually `_`-prefixed argument names mean the arg is unused, but it appears `_ty` is used below; could you rename it to e.g. `ty` or `ctrl_ty` or similar?